finesse/gui/interferometer_monitor.py

```diff
@@ -71,8 +29,7 @@ def __init__(self) -> None:
         self._data_table: list[EM27Property] = []
 
         self._poll_light = LEDIcon.create_poll_icon()
-        self._poll_light._timer.timeout.connect(self.poll_server)  # type: ignore
-        self._poll_light._timer.start(2000)
+        self._poll_light._timer.timeout.connect(self._poll_server)  # type: ignore
```

Comment:
Why you need to ignore this line? Why is mypy complaining?

Not sure if related, but any tribute that starts with `_` should be treated as internal to the class and not be used externally. So, you should not use `self._poll_light._timer`. If this is a legitimate use of `_timer`, then it should not be "private" and the underscore should be removed.

Reply:
The `_timer` attribute is a `QTimer`, and `mypy` and Qt don't seem to get along.

But yes, I've not been consistent with my public and private attributes as the code has evolved -I'll fix this.